docs/content/preview/yugabyte-voyager/diagnostics-report.md

```diff
@@ -90,13 +90,13 @@ To control whether to send diagnostics to Yugabyte, you can use the `--send-diag
 Set the flag as follows:
 
 ```sh
-yb-voyager ... --send-diagnostics=[true|false|yes|no|1|0]
+yb-voyager ... --send-diagnostics [true|false|yes|no|1|0]
 ```
 
 The default is true.
 
 Alternatively, you can use the following environment variable to set a value for this flag globally on the client machine where yb-voyager is running.
 
 ```sh
-export YB_VOYAGER_SEND_DIAGNOSTICS=[true|false|yes|no|1|0]
+export YB_VOYAGER_SEND_DIAGNOSTICS [true|false|yes|no|1|0]
```

Comment:
environment variable should be with `=`, only the argument provided to a command need to be changed to have a space.
`export YB_VOYAGER_SEND_DIAGNOSTICS=[true|false|yes|no|1|0]`